tracetools/src/tp_define.c

```diff
@@ -0,0 +1,17 @@
+// Copyright 2019 Robert Bosch GmbH
```

Comment:
just checking, is this copyright correct?

Reply:
It is a copy indeed.